diff -urN fuse-1.0.0.1a/screenshot.c fuse-1.0.0.1a-patched/screenshot.c
--- fuse-1.0.0.1a/screenshot.c	2010-11-30 06:07:12.000000000 +1000
+++ fuse-1.0.0.1a-patched/screenshot.c	2012-01-18 02:29:06.517348363 +1100
@@ -49,6 +49,7 @@
 #ifdef USE_LIBPNG
 
 #include <png.h>
+#include <zlib.h>
 
 static int get_rgb32_data( libspectrum_byte *rgb32_data, size_t stride,
 			   size_t height, size_t width );
